src/index.ts

```diff
@@ -1,3 +1,61 @@
-import { Server } from './server';
+import * as socketRabbit from 'rabbit-socket.io';
+import * as http from 'http';
+import express from 'express';
+import { router } from './router';
+import { config } from './config';
 
-Server.startServer();
+export interface IMessage {
```

Comment:
Add comments to all functions with their params and return values

Reply:
done